static/src/javascripts/projects/common/modules/commercial/build-page-targeting.js

```diff
@@ -96,12 +91,6 @@ define([
                 vl:      page.contentType === 'Video' ? (Math.ceil(page.videoDuration / 30.0) * 30).toString() : undefined
             }, audienceScienceGateway.getSegments(), criteo.getSegments());
 
-        if (kruxAbParam()) {
-            pageTargets.x = krux.getSegments();
-        } else {
-            pageTargets.a = audienceScience.getSegments();
```

Comment:
Since you are not using audienceScience, do you need to include 'common/modules/commercial/third-party-tags/audience-science'?